Makefile

```diff
@@ -150,7 +150,8 @@ scripts/vendor: scripts/jsonnetfile.json scripts/jsonnetfile.lock.json
 
 install-tools:
 	@echo Installing tools from tools.go
-	@cat tools/tools.go | grep _ | awk -F'"' '{print $$2}' | xargs -tI % go install %
+	@cd tools && cat tools.go | grep _ | awk -F'"' '{print $$2}' | xargs -tI % go get %
```

Comment:
why is this needed?

Reply:
The go.mod in the parent directory will record dependencies if we are not changing to a different directory.

Reply:
It shouldn't be the case if we specify "-mod=readonly" as commented below

Reply:
Yep, that way it works. thanks for the suggestion!

Reply:
You're welcome!